declarations.d.ts

```diff
@@ -0,0 +1,14 @@
+declare module "*.pb" {
```

Comment:
protobuf auto-generates this file based on contents inside `msg.proto`, look at `msg.pb.d.ts` after running:
```
make deno
```
or in case you don't want to build v8 (which takes ~30min)
```
make msg.pb.d.ts